packages/eslint-plugin-fbtee/src/rules/no-unwrapped-strings.tsx

```diff
@@ -0,0 +1,144 @@
+import {
+  createRule,
+  hasFbtParent,
+  isNodeFbt,
+  propName,
+  resolveNodeValue,
+} from '../utils.tsx';
+
+type Options = [{ ignoredWords: Array<string> }];
+
+const attributes = new Set([
+  'title',
+  'placeholder',
+  'alt',
+  'label',
+  'aria-label',
+  'aria-errormessage',
+]);
+
+export default createRule<Options, 'unwrappedString'>({
+  create(context, options) {
+    const ignoredWords = new Set(
+      options[0].ignoredWords.map((s) => s.trim().toLowerCase()),
+    );
+
+    function isIgnoredWord(value: unknown) {
```

Comment:
Could the typing here be improved to only the expected types that will be passed in here? I assume it's mostly `number` or `string`?